Utility/DynamicJsonParser.cs

```diff
@@ -135,6 +137,55 @@ private static dynamic GetDynamic(PSObject obj, string numericalPrefix = "n", bo
                     propertyName = propertyName.ToLowerInvariant();
                 }
 
+                // Nuspec contains xml metadata as a string (only applicable to VSTS feeds)
+                if (propertyName == "nuspec")
+                {
+                    XmlDocument xmlDoc = new XmlDocument();
+                    string xmlString = (string)psProperty.Value;
+
+                    if (xmlString[0] != '<')
+                    {
+                        var strIndex = xmlString.IndexOf('<');
+                        xmlString = xmlString.Substring(strIndex, xmlString.Length - 1);
+                    }
+
+                    xmlDoc.LoadXml(xmlString);
+                    XmlNodeList metadataNodeList = xmlDoc.GetElementsByTagName("metadata");
+                    List<string> propertyList = new List<string>();
+
+                    // Create a list of the child tags of metadata (id, version, etc.)
+                    foreach (XmlNode node in metadataNodeList)
```

Comment:
generally we shouldn't need to parse the XML with regexes, it should be possible to get what we want direct from the xmlDoc object model. Could you include in a comment an example of the expected format of this nuspec tag? That would help explain what the code is doing

Reply:
Here's the contents of nuspec:
"nuspec":"﻿<?xml version=\"1.0\" encoding=\"utf-8\"?>\r\n<package xmlns=\"http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd\">\r\n  <metadata>\r\n    <id>Microsoft.Kusto.Tools</id>\r\n    <version>1.0.6</version>\r\n    <title>Kusto Tools</title>\r\n    <authors>Microsoft</authors>\r\n    <owners>Microsoft</owners>\r\n    <requireLicenseAcceptance>false</requireLicenseAcceptance>\r\n    <licenseUrl>http://kusto.blob.core.windows.net/kusto-nuget/EULA-agreement.htm</licenseUrl>\r\n    <projectUrl>http://aka.ms/kdocs</projectUrl>\r\n    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</iconUrl>\r\n    <description>Kusto Tools</description>\r\n    <releaseNotes>1.0.6: LightIngest refactored</releaseNotes>\r\n    <copyright>Copyright ©  Microsoft Corporation</copyright>\r\n  </metadata>\r\n</package>"

I initially had some issues pulling out the information I needed via xmlDoc, but I just tried again and was successfully able to get everything I need.  I'll push those changes.

Reply:
Thanks - I meant a comment in the actual code so that people looking at the code in future can see it